src/modules/fancyzones/FancyZonesLib/FancyZonesWindowProperties.h

```diff
@@ -27,5 +27,7 @@ namespace FancyZonesWindowProperties
 
     std::optional<size_t> GetTabSortKeyWithinZone(HWND window);
     void SetTabSortKeyWithinZone(HWND window, std::optional<size_t> tabSortKeyWithinZone);
+
+    bool RetrieveFilteringProperties(HWND window);
```

Comment:
You're right. I first thought that it would be better this way in case we add other properties in the future, but keeping them separate would be cleaner.